studies/templates/studies/study_edit.html

```diff
@@ -39,99 +18,96 @@
         {% for message in messages %}
             {% if message.extra_tags != 'user_added' and message.extra_tags != 'user_removed' %}
                 <div id="details" class="alert {{ message.tags }} alert-dismissable">
-                    <a href="#" class="close" data-dismiss="alert" aria-label="close">×</a>
+                    <a href="#" class="close" data-bs-dismiss="alert" aria-label="close">×</a>
                     {{ message }}
                 </div>
             {% endif %}
         {% endfor %}
     {% endif %}
 {% endblock flash %}
```

Comment:
I am inclined to remove the messages block. Did you find it to be necessary in this form?

Reply:
Ah good question. I believe the messages block is used after a study is saved successfully. 
<img src="https://user-images.githubusercontent.com/9041788/223231922-14808644-8528-4f1b-b96d-ee8a7c92aa85.png" width="50%"/>

And the change here (`data-bs-dismiss`) adds the close (X) button on the right side, and apparently some functionality: https://getbootstrap.com/docs/5.0/components/alerts/#dismissing

Reply:
There's a message tag in the base template.  Does that not do the same thing?

Reply:
Ahh sorry, missed that. I'll check now.

Reply:
Yep you're right, I'll take it out. 
Do I need to replace it with `{% bootstrap_messages %}` as in here:
https://github.com/lookit/lookit-api/blob/f7cbb535bc1b95346403378e47d0c8600659456b/studies/templates/studies/study_form.html#L29
and here:
https://github.com/lookit/lookit-api/blob/f7cbb535bc1b95346403378e47d0c8600659456b/studies/templates/studies/study_detail.html#L12

It seems to work without it..?

Reply:
Those should be removed as well.  Thanks for the clean up.

Reply:
Done here: 5b882143c2807c71baf6d7aac7bb39399a558cf0 How's that?